### DISCUSSION TITLE
ETA on next release?

### DISCUSSION THREAD

**Discussion by BayleeRichards** [dotnet/command-line-api#2203] [Q&A]
We are really excited to get started using some of the goodies developed as of recent. There was a discussion at #2190 a few weeks ago stating an upcoming merge will be happening soon, is there any ETA on a release date? If there is none, could someone please point us to a stable daily release build that makes use of the new syntax, hosting and functional tab completion?

Thanks in advance, love the project!

**Comment by mickelsonmichael** [comment]
Would like to bump this thread, it's been quite a few months now, and the API to the package has changed significantly and it is no longer quite a convenient reference. There's also features in the new version that I'd love to use and it's frustrating to see that the feature exists but I can't access it yet 😅

Thanks for all the work though! Really nice library so far

**Reply by rrickgauer** [reply]
I am also curious as to when this will be released

**Comment by rrickgauer** [comment]
Bump

**Comment by vmelamed** [comment]
I also would love to see some movement here. IMO, this is a better CLI package consistent with the rest of the "CLI world" and I'd prefer to use it but the "preview" status is a problem.

**Reply by bhehe** [reply]
Same for me; The fact it's labeled pre-release goes against our general practice of not allowing anything to go to Prod that is using pre-release packages.

**Comment by elgonzo** [comment]
Just as an FYI to all observing this discussion thread, this released today/yesterday: https://github.com/dotnet/command-line-api/issues/2338
